frontend/src/components/Button/ButtonLink.tsx

```diff
@@ -0,0 +1,17 @@
+import { Link } from 'react-router-dom';
```

Comment:
Please add a bit documentation for each component which might be reused (CTA etc. is okay to not have much documentation).